2 - RTB/Documentazione interna/Verbali/24-01-17/24-01-17.typ

```diff
@@ -0,0 +1,59 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "17/01/24",
+  subTitle: "Meeting di pianificazione RTB",
+  docType: "verbale",
+  authors: (
+    "Todesco Mattia",
+  ),
+  timeStart: "10:00",
+  timeEnd: "11:00",
+);
+
+= Ordine del giorno
+
+- Valutazione progresso generale;
+- verifica avanzamenti per colloquio RTB;
+- candidatura RTB.
+
+
+== Valutazione progresso generale
+
+Ogni membro del gruppo ha esposto lo stato di avanzamento delle proprie attività assegnate.
+
+=== Analisi dei Requisiti
+
+Il documento è completo e revisionato, sono state apportate le ultime modifiche ed ora è pronto per la presentazione RTB.
+
+=== Comunicazioni con il Proponente
+
+Il gruppo ha inviato una mail al Proponente corredata di video dimostrativo sul funzionamento del PoC. 
+Il gruppo resta in attesa di risposta da parte dello stesso.
+
+== Verifica avanzamenti per colloquio RTB
+
+=== Presentazione power point
+
+La presentazione prodotta è stata revisionata da tutti i membri del gruppo. I contenuti sono stati accettati con solo alcune modifiche minori da segnalare, mentre il tema della stessa risulta ancora "acerbo" e pertanto va migliorato.
+
+=== Disponibilità per colloquio RTB
+
+Vista l'imminente sessione d'esami il gruppo ha unanimamente concordato delle date in cui il colloquio RTB non può essere svolto causa impegni universitari inderogabili. Le date sono:
+- 22/01/24 di mattina;
+- 24/01/24 tutto il giorno;
+- 26/01/24 di mattina.
+
+=== Lettera di presentazione
+
+La lettera di presentazione non risulta ancora redatta.
+
+== Candidatura RTB
+
+Il gruppo si ritiene pronto per effettuare la candidatura al colloquio RTB entro i termini precedentemente stabiliti, cioè entro il 19/01/24. Essa verrà inviata non appena terminata la lettera di presentazione richiesta e dopo aver migliorato il tema della presentazione power point.
+
+== To Do
+
+- Redigere lettera di presentazione;
```

Comment:
La prima parola degli elementi dell'elenco puntato va in maiuscolo, in coerenza con gli altri verbali